### DISCUSSION TITLE
Out of date documentation?

### DISCUSSION THREAD

**Discussion by neoacevedo** [fleaflet/flutter_map#1959] [General]
https://docs.fleaflet.dev/v/v3/usage/options/recommended-options

I think the documentation is currently out of date. For instance, for the `MapOptions`, you are using the `center` property which isn't available, instead the `MapOptions` is using `initialCenter` as replacement.

**Comment by JaffaKetchup** [comment]
Hi @neoacevedo,
That appears to link to the documentation for v3. You can change the version using the selector in the top left or menu. Does that bring you to the documentation you need?

**Comment by neoacevedo** [comment]
Yes, you're right. My apologies.
